neo/rawio/elanrawio.py

```diff
@@ -139,12 +147,12 @@ def _parse_header(self):
         self._raw_signals = self._raw_signals[:, :-2]
 
         # triggers
-        with open(self.filename + '.pos', mode='rt', encoding='ascii', newline=None) as f:
+        with open(self.posfile, mode='rt', encoding='ascii', newline=None) as f:
             self._raw_event_timestamps = []
             self._event_labels = []
             self._reject_codes = []
             for l in f.readlines():
-                r = re.findall(r' *(\d+) *(\d+) *(\d+) *', l)
+                r = re.findall(r' *(\d+)\s* *(\d+)\s* *(\d+) *', l)
```

Comment:
Does it work with older file or does it break something ?
Do you have a file we could put in https://gin.g-node.org/NeuralEnsemble/ephy_testing_data to ensure that this work in all cases ?

Reply:
I think it should work with older files. For example, here's how the regexp works on the file you sent me ([link](https://regex101.com/r/07iYqf/1)) . Then, here's a more recent new pos file ([link](https://regex101.com/r/EVwW53/1)). In the second one, you can try to replace with the old regex pattern and you should see that event are not correctly found (maybe because new pos files uses tab...? not sure)